internal/contour/route.go

```diff
@@ -143,6 +143,11 @@ func (v *routeVisitor) visit(vertex dag.Vertex) {
 					routes = append(routes, &envoy_api_v2_route.Route{
 						Match:  match,
 						Action: envoy.RouteRoute(route),
+
+						RequestHeadersToAdd:     envoy.HeaderValueList(route.SetRequestHeaders, false),
```

Comment:
nit: Should we match these names with `Set`?

Reply:
This is Envoy's API, so it's the one thing we can't change.  The `false` parameter is what toggles append vs. set semantics.

Reply:
Ohhhh ok you're right, sorry I overlooked that. =)